src/app/help/en/faq.md

```diff
@@ -512,7 +512,7 @@ Currently steemit.com supports English, Spanish, Russian, French, Italian, Korea
 
 ## <span id="What_can_users_post_to_Steemit">What can users post to Steemit?</span>
 
-Steem is an open platform meant to host and welcome any legal content. Users can post anything they want, whether it be phrases, quotes, blogs, anecdotes, photos, videos, memes, songs, and more. Be creative!
+Steem is an open platform meant to host and welcome any legal content, that compiles with our [terms of service](/tos.html). Users can post anything they want, whether it be phrases, quotes, blogs, anecdotes, photos, videos, memes, songs, and more. Be creative!
```

Comment:
Did you mean "complies" as opposed to "compiles?"

Reply:
Also -- please remove the comma